README.md

```diff
@@ -22,6 +25,10 @@ cv.onRuntimeInitialized = () => {
 
 ```
 
+## Requirements
+
+It is necessary that opencv is installed on your computer, and the paths configured (windows) correctly as instructed by [OpenCV.org](https://opencv.org/)
```

Comment:
I don't think this is the case of OpencvJs, you don't even need the compiled WASM because it's packaged with this project.

Reply:
I understand! I try to unninstall my opencv libraries, test again, and report tests.

Reply:
I remove this block in the next commit.
I left only the first initial